loss-detection-app/extensions/add_frame_id.py

```diff
@@ -0,0 +1,32 @@
+'''
+* Copyright (C) 2019-2020 Intel Corporation.
```

Comment:
can you update the copyright year to 2022

Reply:
Done